agent/internal/container/container_api_test.go

```diff
@@ -215,3 +218,134 @@ func TestContainer(t *testing.T) {
 		})
 	}
 }
+
+func TestContainerStatus(t *testing.T) {
+	log.SetLevel(log.TraceLevel)
+
+	t.Log("building client")
+	rawCl, err := dclient.NewClientWithOpts(dclient.WithAPIVersionNegotiation(), dclient.FromEnv)
+	require.NoError(t, err)
+	defer func() {
+		if cErr := rawCl.Close(); cErr != nil {
+			t.Logf("closing docker client: %s", cErr)
+		}
+	}()
+	cl := docker.NewClient(rawCl)
+
+	tests := []struct {
```

Comment:
this one is debatable but could be simpler to avoid parameterizing the test since we only have one but either way is fine.

Reply:
I went ahead and took this out because it was overkill for what it currently is, and in the event that we'll need to run multiple tests through this setup, it's just a matter if adjusting it to match the test function above it 😄